src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -838,7 +838,8 @@
 
      The options are:
 
-     `:router-targets` - (REQUIRED) A *vector* of ui components that are router targets. The first one is considered the \"default\".
+     `:router-targets` - (REQUIRED) A *vector* of ui components that are router targets. The first one is considered the \"default\"
+     (purely for the purpose if initial state; you always need to explicitly route to a particular target).
```

Comment:
Sorry, didn't see this PR till now for some reason. You have a typo: "if" should be "of"

Reply:
@awkay No problem! Fixed the typo now.